addons/survey/static/src/js/survey_session_leaderboard.js

```diff
@@ -252,28 +241,9 @@ publicWidget.registry.SurveySessionLeaderboard = publicWidget.Widget.extend({
     },
 
     /**
-     * After displaying the score for the current question, we sum the total score
```

Comment:
Seeing the care put into all of this, and that no one really complained about it before, I wonder if we shouldn't apply our changes in 18.3 instead (I suspect there is little to change from this PR), and perhaps backport later if people do have issues... What do you think, @pyduf ?